src/components/KeyValue/PKeyValue.vue

```diff
@@ -94,9 +95,6 @@
   @apply
   text-xs
   leading-4
-  flex
-  flex-col
-  gap-y-1
```

Comment:
@pleek91 I'm not overly concerned here. But these flex styles tell me that if there were multiple items within the label, they'd have column styling with a gap. Removing them would probably revert them to inline or something.

Reply:
Okay so these are part of the BEM modifier class `p-key-value--alt`. It's intent is to modify the `p-key-value` block class. But that block class already has these styles so they were being applied twice (I noticed this in the dom when testing). No functional change of styles is being made here. Because this modifier class is not used or intended to be used on its own and these styles are present in the primary block level class. 

I can go into more detail about BEM class semantics sometime if you still have questions about this

Reply:
Ah ok. I never looked at the root class.

No need for a BEM review, haha, even though I have strayed from the path many times before, thank you though.